src/services/mockMoviesData.js

```diff
@@ -1,32 +1,42 @@
 const movies = [
     {
+        id: 1,
         title: 'Movie 1'
```

Comment:
I decided to add "id" parameter to mockData instead of using the index from map()